app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -101,6 +100,8 @@ open class FenixApplication : Application() {
             ExperimentsMetrics.activeExperiment.set(branchName)
         }
 
+        ExperimentsManager.initETPExperiment(this)
```

Comment:
Tiny nit: it's possible that we have a different practice in this project, but afaik standard style guides for casing specify that acronyms should be lowercased after the first letter.  This prevents weird edge cases like `XMLHTTPRequest`, which should be `XmlHttpRequest`.

https://developer.android.com/kotlin/style-guide#camel_case
(I shamelessly stole my example from this link)